po/ru.po

```diff
@@ -1065,40 +1075,59 @@ msgid ""
 "  mutexes) and async/await concurrency (cooperative multitasking using\n"
 "  futures)."
 msgstr ""
+"* [Android](android.md): рассчитанный на полдня курс по использованию Rust для разработки на\n"
+"платформе Android (AOSP).\n"
+"Сюда входит взаимодействие с C, C++ и Java.\n"
+"* [Bare-metal](bare-metal.md): однодневное занятие по использованию Rust для bare-metal\n"
+"  (embedded)разработки. Охватываются как микроконтроллеры, так и прикладные процессоры.\n"
+"* [Concurrency](concurrency.md): полный день занятий по изучению конкуретности в Rust.\n"
+"Мы рассмотрим как классическую конкурентность\n"
+"(вытесняющее планирование с использованием потоков и мьютексов),\n"
```

Comment:
Afaik "concurrency" is usually translated as "параллельность", not "конкурентность"?

Reply:
I was trying to follow [this dictionary](https://github.com/rust-lang-ru/dictionary). It suggests to translate `concurrency` as `конкурентность`. IMO it's more accurate translation, than `параллельность`, since these two are not the same. 
Again, it's just my opinion and if we decide to translate it as `параллельность` I'm more than happy to make changes.
@ramosian-glider WDYT ?

Reply:
I think the canonical term for concurrency in Russian is "параллелизм" or "параллельная обработка".

Granted, I haven't read a cs book in Russian in a while, but I'm not even sure if I would understand what "конкурентность" means here.  Multitran dictionary [doesn't have](https://www.multitran.com/m.exe?s=concurrency&l1=1&l2=2) "конкурентность" for concurrency either.

Though I do understand the value of having a consistent translation between TRB and this tutorial.

Reply:
+1 on параллельная обработка 'конкурентность' means competitiveness

Reply:
nit: let's try not to mix parallelism and concurrency - they are 2 independent concepts

Reply:
может "многозадачность"?

Reply:
`многозадачность` (multitasking ?) looks interesting, although I am at a loss to say whether it means the same thing as `concurrency` (conceptually).

Reply:
Либо оставить как есть, конкуррентность. Но точно не параллелизм (я преподаю курс по конкарренси, в терминах есть разница).